src/rgw/rgw_op.cc

```diff
@@ -702,7 +702,7 @@ int rgw_build_bucket_policies(rgw::sal::RGWRadosStore* store, struct req_state*
  * Returns: 0 on success, -ERR# otherwise.
  */
 int rgw_build_object_policies(rgw::sal::RGWRadosStore *store, struct req_state *s,
-			      bool prefetch_data)
+			      bool prefetch_data, void *rangeit)
```

Comment:
why is this a void pointer?  shouldn't it be prefetch_range* (or prefetch_range& or even prefetch_range, as it's probably not a big deal to copy one)?

Reply:
Fixed it, thank you for your input, it helped me